apps/noise_aware.cpp

```diff
@@ -26,14 +27,16 @@ int main(int argc, char** argv) {
         ("simulate_file", "simulate a quantum circuit given by file (detection by the file extension)", cxxopts::value<std::string>())
         ("step_fidelity", "target fidelity for each approximation run (>=1 = disable approximation)", cxxopts::value<double>()->default_value("1.0"))
         ("steps", "number of approximation steps", cxxopts::value<unsigned int>()->default_value("1"))
-
+        // Parameters for noise aware simulation
         ("noise_effects", "Noise effects (A (=amplitude damping),D (=depolarization),P (=phase flip)) in the form of a character string describing the noise effects (default=\"APD\")", cxxopts::value<std::string>()->default_value("APD"))
         ("noise_prob", "Probability for applying noise (default=0.001)", cxxopts::value<double>()->default_value("0.001"))
-        ("confidence", "Confidence in the error bound of the stochastic simulation (default= 0.05)", cxxopts::value<double>()->default_value("0.05"))
-        ("error_bound", "Error bound of the stochastic simulation (default=0.1)", cxxopts::value<double>()->default_value("0.1"))
-        ("stoch_runs", "Number of stochastic runs. When the value is 0 the value is calculated using the confidence, error_bound and number of tracked properties. When the value is -1, the deterministic simulator is started. When the value is -2 the old method for deterministic noise application is used) (default = 0)", cxxopts::value<long>()->default_value("0"))
-        ("properties", R"(Comma separated list of tracked properties. Note that -1 is the fidelity and "-" can be used to specify a range.  (default="-3-1000"))", cxxopts::value<std::string>()->default_value("-3-1000"))
-    ;
+        ("noise_prob_t1", "Probability for applying amplitude damping noise (default=2 x noise_prob)", cxxopts::value<double>()->default_value("-1"))
+        ("noise_prob_multi", "Noise factor for multi qubit operations (default=2)", cxxopts::value<double>()->default_value("2"))
+        ("unoptimized_sim", "Use unoptimized scheme for stochastic/deterministic noise-aware simulation")
+        ("stoch_runs", "Number of stochastic runs. When the value is 0, the deterministic simulator is started. (default = 0)", cxxopts::value<long>()->default_value("0"))
```

Comment:
Should be `unsigned long` (better yet `std::size_t`), shouldn't it?

Reply:
Changed the code as suggested